### PR TITLE
SWIK-960 periodically clean collection

### PR DESCRIPTION
**Pull request by TBoonX**
See commit notes

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,18 +1,41 @@
...
+# ----------------------- #
+#   Installation NodeJS   #
+# ----------------------- #
 
 ADD ./application/package.json ./
 RUN npm install --production
 
 ADD ./application/ ./
 
+EXPOSE 3000 80
```
Comment (rmeissn) [inline Dockerfile]: Why does this service exposes port 3000?

File: supervisord.conf [hunk 2]
```diff
@@ -0,0 +1,16 @@
+[supervisord]
```
Comment (rmeissn) [inline supervisord.conf:1]: Are cron logs (by your script) also logged as part of stdout? --> I want to see these as part of `docker logs [container_name]`

File: application/database/cleanup.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const helper = require('./helper'),
+  collectionName = 'oauthprovideruserdata',
+  millisNeededForTimeout = 2*86400*1000;  //2 days
```
Comment (rmeissn) [inline application/database/cleanup.js]: This is not a timeout. It's more like expired or out of date.

**Review by rmeissn — COMMENTED** [review]
Haven't tested functionality yet.

File: Dockerfile (hunk 1 above)
Comment (TBoonX) [reply Dockerfile]: fault

File: supervisord.conf (hunk 2 above)
Comment (TBoonX) [reply supervisord.conf:1]: no it does not ...

File: application/database/cleanup.js (hunk 3 above)
Comment (TBoonX) [reply application/database/cleanup.js]: you are right

**Pull request merged**